frontend/src/views/SettingWorkspaceGeneral.vue

```diff
@@ -1,14 +1,97 @@
 <template>
   <div class="mt-2 space-y-6 divide-y divide-block-border">
-    <div v-if="allowEdit" class="pt-5 flex justify-end">
-      <button
-        type="button"
-        class="btn-primary"
-        :disabled="!allowSave"
-        @click.prevent="doSave"
-      >
-        {{ $t("common.update") }}
-      </button>
+    <div>
+      <FeatureAttention
+        v-if="!hasBrandingFeature"
+        custom-class="mt-5"
+        feature="bb.feature.branding"
+        :description="$t('subscription.features.bb-feature-branding.desc')"
+      />
+      <div class="px-4 py-6 md:flex">
+        <h1 class="text-left text-2xl font-bold w-1/3">
+          {{ $t("settings.general.workspace.branding") }}
+        </h1>
+        <div class="flex-1 md:px-5">
+          <!-- <div class="mb-5">
+            <p class="mb-3">
+              {{ $t("settings.general.workspace.display-name") }}
+            </p>
+            <input
+              v-model="state.displayName"
+              pattern="[a-z]+"
+              type="text"
+              name="org-display-name"
+              class="shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full border-gray-300 rounded-md"
+              placeholder="Enter a friendly name for your workspace"
+            />
+          </div> -->
```

Comment:
Will remove the code here of we're not planning to support customizing the display name.